File: Number_Guessing_Game/Game.cpp
#include <bits/stdc++.h>
using namespace std;


int main(){
cout<<"Hello, Today you will guess the generated number! Let's GO!"<<endl;
    srand(time(0));
    int ran = rand() % 101;
    int n;
    int att = 5;
    while(att--){
        cin>>n;
        if(n<ran){
            cout<<"BIGGER"<<endl;
        }
        
        else if(n>ran){
            cout<<"SMALLER"<<endl;
        }
        
        else{
            cout<<"YOU WON!"<<endl;
            return 0;
        }
    }
        cout<<"YOU LOST YOUR NUMBER WAS: "<<ran<<endl;
}

File: Number_Guessing_Game/Game.py
import random

print("Hello, Today you will guess the generated number! Let's GO!")
ran = random.randint(0, 100)
att = 5

while att > 0:
    n = int(input())
    if n < ran:
        print("BIGGER")
    elif n > ran:
        print("SMALLER")
    else:
        print("YOU WON!")
        break
    att -= 1
else:
    print(f"YOU LOST YOUR NUMBER WAS: {ran}")
